python/src/trezorlib/cardano.py

```diff
@@ -185,3 +187,19 @@ def _create_change_address_output(output) -> messages.CardanoTxOutputType:
         )
 
     raise ValueError("Unknown address type")
+
+
+def create_certificate(certificate) -> messages.CardanoTxCertificate:
+    # todo: GK - some validation, checks
```

Comment:
This looks too unfinished. Perhaps separate all CLI stuff into additional PRs --- the main functionality is supposedly checked by unit tests (and if it is not, we should add some within the PR where the functionality is added).

Reply:
I finished it in this PR. It's not that much changes and it's kind of needed to test the functionality manually.

Tests are added in the next PR.